dimod/generators/anti_crossing.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2020 D-Wave Systems Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+# =============================================================================
+
+from dimod import BinaryQuadraticModel
+__all__ = ['anti_crossing_clique', 'anti_crossing_loops']
+
+
+
+def anti_crossing_clique(num_qubits):
+    """Anti crossing problems with a single clique.
+
+    Given the number of qubits, the code will generate a clique of
```

Comment:
Maybe we should use "variables/interactions" rather than "qubit/couplers". Though I suppose this is mostly a useful in the context of QPUs. Could go either way.